packages/mdc-toolbar/README.md

```diff
@@ -0,0 +1,111 @@
+# MDC Toolbar
+
+MDC Toolbar provides an `mdc-toolbar` class for easily styling a RTL-aware
+toolbar. The toolbar scrolls with content by default, but supports fixed on
+top as well.
+
+
+## Installation
+
+```
+npm install --save @material/toolbar
+```
+
+
+## Usage
+
+Wrap the items with `mdc-toolbar` class in following way:
+
+```html
+<div class="mdc-toolbar">
+  <div class="mdc-toolbar__section">...</div>
+</div>
+```
+
+### Fix to top
+
+By default, toolbar scroll with the content, you can add `mdc-toolbar_fixed` to
+fix toolbar to the top of screen.
+
+**Body margin required**
```

Comment:
I'm not so sure how I feel about this approach. Having to wrap a toolbar in a containing element imposes an implicit DOM structure onto the end user which they must follow, and assumes that all of the contents within that element will need to be pushed down. I was thinking instead of making this a standalone `mdc-toolbar-fixed-adjust` class which would simply add a `margin-top` property to whichever element it was added to that could offset it by the height of the toolbar. This provides a lot more flexibility since it means that the toolbar doesn't have to be contained within a given element for the UI to function properly. Consider:

```html
<!-- App Header section -->
<header class="mdc-toolbar mdc-toolbar--fixed">...</header>
<main class="mdc-toolbar-fixed-adjust">
  ...
</main>
```

In the above example, if we needed a wrapper element than `<header>` would have to share a common ancestor with `<main>`, which may not make that much sense semantically.